tests/suite/fixtures/ic_fixtures.py

```diff
@@ -267,7 +267,7 @@ def crd_ingress_controller_with_dos(
 
         print("------------------------- Create syslog svc -----------------------")
         src_syslog_yaml = f"{TEST_DATA}/dos/dos-syslog.yaml"
-        log_loc = f"/var/log/messages"
+        f"/var/log/messages"
```

Comment:
is this needed at all?

Reply:
Good catch. no its not, I have removed it now. Thanks